controllers/pvc.go

```diff
@@ -174,7 +174,12 @@ func (*MySQLClusterReconciler) needResizePVC(cluster *mocov1beta2.MySQLCluster,
 		case i == 0: // volume size is equal
 			continue
 		case i == 1: // volume size is greater
-			return nil, false, fmt.Errorf("failed to resize pvc %q, want size: %s, deployed size: %s: %w", pvc.Name, wantSize, deployedSize, ErrReduceVolumeSize)
```

Comment:
Please update the following test in `controllers/pvc_test.go`.
https://github.com/cybozu-go/moco/blob/v0.16.1/controllers/pvc_test.go#L157-L163

Also, add a new test case. 
A MySQLCluster has 2 volume claim templates. One expands the volume size, and the other reduces it.

Reply:
I added tests.
Also, there was a problem with the test code, which has been corrected.
https://github.com/cybozu-go/moco/pull/552/commits/32b8fbbd9cc1b4ea2b024e04f6f1a82a57282d2f